compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -761,6 +762,20 @@ static LogicalResult setTransformDialectConfig(func::FuncOp entryPoint,
   return setTranslationInfo(entryPoint, translationInfo);
 }
 
+static bool isMatvecLike(linalg::LinalgOp linalgOp) {
+  // TODO: Allow for matvec with fused dequantization.
+  if (!linalg::isaContractionOpInterface(linalgOp))
+    return false;
+
+  if (linalgOp.getNumParallelLoops() != 2)
+    return false;
+
+  if (linalgOp.getNumReductionLoops() > 2)
+    return false;
```

Comment:
Is there a specific reason it does not support multi-reduction dim like the dequant matvec? IIRC it lowers similarly to one reduction dim anyways as long as they are fastest dimensions.

Reply:
Quinn made a similar comment, see https://github.com/openxla/iree/pull/15941#discussion_r1427567016. I'm going to close this thread to deduplicate.